src/components/Preson.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const partner = (sex === 'f') ? 'husband' : 'wife';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        <p className="Person__age">
+          {age
+            ? `I am ${age}`
+            : ''
+          }
+        </p>
```

Comment:
Use conditional rendering for the whole <p> tag. No need to create empty html-elements.

```
{age && (
  <p className="Person__age">
     {`I am ${age}`}
  </p>
)}
```

Reply:
Fixed